bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/NetatmoBridgeHandler.java

```diff
@@ -210,6 +210,11 @@ private String getApiScope() {
             scopes.add("write_camera");
         }
 
+        if (configuration.readPresence) {
```

Comment:
Just a comment, I see the scope "write_camera" is used for the Welcome camera. I don't remember what it is its role. But are you sure that it is not required for the Presence ?

Reply:
Yes, write_camera isn't existing for the Presence camera. This is documented here: https://dev.netatmo.com/apidocumentation/security#scopes